pkg/providers/core.go

```diff
@@ -39,6 +39,7 @@ func CoreConfigProviders() di.Option {
 		di.Provide(server.NewMetricsConfig, di.As(new(environments.ConfigModule))),
 		di.Provide(workers.NewReconcilerConfig, di.As(new(environments.ConfigModule))),
 		di.Provide(auth.NewContextConfig, di.As(new(environments.ConfigModule))),
+		di.Provide(auth.NewAdminAuthZConfig, di.As(new(environments.ConfigModule))),
```

Comment:
let's also register this as ServiceValidator as per https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1293/files#r980230464 comment